packages/console/src/console-history/ConsoleHistoryResultErrorMessage.jsx

```diff
@@ -99,7 +113,7 @@ class ConsoleHistoryResultErrorMessage extends PureComponent {
             <button
               type="button"
               onClick={this.handleToggleError}
-              className="error-btn-link"
+              className={arrowBtnClasses}
```

Comment:
This looks much better! Only issue is that the button doesn't highlight when you are hovering over it itself.

Could add this here:
```
            onMouseEnter={this.handleMouseEnter}
            onMouseLeave={this.handleMouseLeave}
```
Or just add a `error-message.btn:hover` styling as well